src/Hintman/Suggestion/Core.hs

```diff
@@ -13,18 +13,19 @@ data Line = Line
     } deriving (Eq, Show)
 
 data ChangeType = Edit | Delete
+  deriving (Eq, Show)
```

Comment:
Just noticed that these are introduced in https://github.com/kowainik/hintman/pull/31, to remove after rebase